activerecord/lib/active_record/testing/query_assertions.rb

```diff
@@ -5,29 +5,65 @@ module Assertions
     module QueryAssertions
       # Asserts that the number of SQL queries executed in the given block matches the expected count.
       #
+      #   # Check for exact number of queries
       #   assert_queries(1) { Post.first }
       #
+      #   # Check for any number of queries
+      #   assert_queries { Post.first }
+      #
       # If the +:matcher+ option is provided, only queries that match the matcher are counted.
       #
       #   assert_queries(1, matcher: /LIMIT \?/) { Post.first }
       #
-      def assert_queries(expected_count, matcher: nil, &block)
+      # If the +:ignore_none+ option is provided, any queries (including transaction and schema related)
+      #   that match the matcher are counted.
+      #
+      #   assert_queries(ignore_none: true) { Post.columns }
+      #
+      def assert_queries(expected_count = nil, matcher: nil, ignore_none: false, &block)
```

Comment:
I really don't like `ignore_none` as a parameter name, as it's a double negation.

Also I wonder that the use case is?